session.go

```diff
@@ -184,6 +189,21 @@ func (s *Session) Prepare(ctx context.Context, content string) (Query, error) {
 	return Query{}, fmt.Errorf("prepare failed on all nodes, details: %v", resErr)
 }
 
+func (s *Session) AwaitSchemaAgreement(ctx context.Context) error {
```

Comment:
I think it should be possible to await schema agreement until it happens without setting an explicit timeout. Making a rule that it will do that if timeout is negative will also be confusing as:
- AutoAwaitSchemaAgreementTimeout < 0 = don't wait for agreement
- AwaitSchemaAgreement(..., timeout) < 0 = wait for agreement indefinitely

Reply:
I was more thinking about removing the possibility to await schema agreement indefinitely - isn't it always a good idea to wait with timeout? And no application (?) would be OK with hanging on schema agreement?